Continuación de la página de inicio

AlexCasal opened the pull request:
Quitar lo de 'Join the crew', HECHO,
Hacer que desde la página de inicio se haga un scroll y salgan las FAQs y algo más de info, HECHO,
Quitar el sold out del ROOT, HECHO

AlexCasal committed:
Continué la página index con los huecos para los sponsors, los collaborators y metí las FAQs

danicallero committed:
feat: switch to english

AlexCasal committed:
Deleted package-lock.json

AlexCasal committed:
Installed pnpm

danicallero committed:
Merge branch 'pr/4' into cambios-alex

danicallero committed:
fix: update FAQ descriptions

AlexCasal committed:
Separate by components

jorgeteixe commented:
Yo esperaría antes de mergear esta PR a hacer una review entre varios, hablarlo todo en reunión etc. así hacemos una publicación coordinada con toda la infromación a la vez.

jorgeteixe committed:
chore: add CONTRIBUTING file

jorgeteixe commented:
Acabo de modificar la rama a la que se hace la pull request, mañana lo podemos comentar en la reunión pero vamos a centralizar los cambios en una rama `develop` y luego hacemos una release conjunta con todo lo que necesitemos de manera coordinada desde `develop` a `main`.

AlexCasal committed:
Added Gpul logo on footer and alinged sponsors width

AlexCasal committed:
Rounder borders and back button on sponsors page

danicallero committed:
style: update footer colors per brand guidelines

danicallero committed:
feat: add Markdown files and layout for Code of Conduct, Privacy Policy, and Terms and Conditions pages

danicallero committed:
style: remove bold formatting from section headers

danicallero committed:
style: add rounded corners to sponsorship form button

danicallero committed:
feat: improve FAQ section with better layout and animations

danicallero committed:
fix: adjust FAQ layout to balance column heights

AlexCasal committed:
added background retro and fixed video margin on ultraWhide resolutions

danicallero committed:
chore: rename markdown pages

danicallero committed:
feat: add background to markdown layouts

danicallero committed:
feat: create InfoLayout and add information and rules pages

danicallero committed:
fix: improve footer layout and spacing on mobile

danicallero committed:
fix: adjust spacing, format and layout for collaborators and sponsors sections

danicallero committed:
feat: add schedule page and data

danicallero committed:
style: update event times visualization

danicallero committed:
style: update end indicator color to red for better contrast

danicallero committed:
fix: add aria-label to social media links for accessibility

danicallero committed:
Merge branch 'gpul-org:main' into main

danicallero committed:
Merge pull request #1 from danicallero/main

componente de horario

AlexCasal committed:
feat: create Info.astro, a caraousel with info

danicallero committed:
refactor: improve layout and styling for collaborators and sponsors sections

- moved base element styles to a constant to reduce visual clutter
- improved mobile layout for better responsiveness

danicallero committed:
fix: correct typos

danicallero committed:
feat: add disclaimer about schedule changes and link to Discord for updates

- Also commented out placeholder data for schedule

danicallero committed:
fix: adjust image sizes and spacing

danicallero committed:
feat: add gradient after landing video

danicallero committed:
feat: improve InfoCarousel component

- Moved carousel to a component
- Added light animations on each switch
- Added buttons linking to info sections

AlexCasal committed:
fix: change carousel for a step guide

danicallero committed:
feat: Add logo to ShowStart component

danicallero committed:
chore: add approximate activities based on hack2025

danicallero committed:
feat: add ImageGallery component with carousel functionality

- Carousel includes an ambient effect surounding the component that slightly animates for an organic and visually appealing feel

danicallero commented:
pinky prometo que ese force solo ha pisado mi trabajo 😇

danicallero committed:
feat: add ImageGallery component with images to index page

danicallero committed:
fix: unify drop shadow size across resolutions

Remove the CSS drop shadow from the logo in 'ShowStart' in favor of a traced-shadow SVG . This ensures consistent shadow offset in different screen sizes

danicallero committed:
chore: reorganize repo for multilanguage

danicallero committed:
refactor: tweak layouts and components for better styling and responsiveness

- Replaced global filters with tiled SVG pattern backgrounds in InfoLayout.astro and MarkdownLayout.astro
- Added viewport height fixes for iOS Safari in Layout.astro
- Updated index.astro to use SVG background and improved layout structure

danicallero committed:
fix: adjust layout and styling for collaborator logos

AlexCasal committed:
fix: coming soon efecto on Sponsors component

danicallero committed:
fix: adjust styles for caution tape overlay

prevents text clipping out of frame

danicallero committed:
chore: remove retreat and fosdem images from carousel

danicallero committed:
fix: correct layout paths to resolve build error

blyxyas commented on src/pages/terms.md:
```diff
@@ -0,0 +1,94 @@
+- **Participant**: Any person who, upon accepting these T&C, registers for and takes part in the Event in any of the established modalities.
+- **Sponsor**: A collaborating entity that, depending on its sponsorship tier, may propose challenges and/or grant specific prizes to Participants.
+- **Challenges**: Technical projects or tasks set by the Organizer and/or Sponsors, to be developed exclusively during the Event.
+- **General Prize**: A prize awarded by the Organizer to the best free software project, based on the criteria detailed in [Premio CIXUG](http://premio.cixug.gal).
```
This page is under a n8n login screen. Being that the terms and conditions of a page are usually public, either changing the link to a static, plain-text file, _or_ just hosting the criteria under this repository would be the most sensible thing to do.

danicallero replied:
Thanks for the heads up!!!

danicallero committed:
feat: add animated down arrow to ShowStart component

danicallero committed:
fix: correct links in terms document

blyxyas commented:
Pregunta: Porque estais desarrollando todo el trabajo en una PR conjunta, en vez de en otra branch? Las PRs no tienen por qué ir todas a master/main, una PR se puede hacer a cualquier branch (y así podriais incorporar CI/CD y un sistema de reviews)

migueldeoleiros commented:
> Pregunta: Porque estais desarrollando todo el trabajo en una PR conjunta, en vez de en otra branch? Las PRs no tienen por qué ir todas a master/main, una PR se puede hacer a cualquier branch (y así podriais incorporar CI/CD y un sistema de reviews)

Hola @blyxyas, gracias por el feedback.
El desarrollo lo estamos haciendo en abierto, pero no todos en el equipo tienen experiencia en git, es un proyecto de voluntarios para sacar la web a tiempo para el evento.

danicallero committed:
chore: update logo source to avoid astro tags

danicallero committed:
feat: add registration link and button

danicallero committed:
feat: add multilingual support

danicallero committed:
fix: update registration status and ECTS credits information in FAQ

danicallero committed:
fix: update registration link to use full URL

danicallero committed:
chore: update meta image for improved branding

danicallero committed:
feat: add About section, update styles for links

danicallero committed:
feat: update personal data descriptions in privacy policy

jorgeteixe reviewed (approved):
LGTM! Great work 🚀
